ansibleflow/app.py

```diff
@@ -50,13 +50,29 @@ def setup_argument_parser():
     )
 
     run_parser = subparsers.add_parser('run')
-    run_parser.add_argument('--env', type=str, default='default')
     run_parser.add_argument(
         'run_action',
-        nargs='*',
+        type=str,
         default=True,
         help=argparse.SUPPRESS
     )
+    run_parser.add_argument(
+            '--env',
+            type=str,
+            default='default')
+    run_parser.add_argument(
+            "-v",
+            "--verbosity",
+            action="count",
+            default=0,
+            help="increase output verbosity")
+    run_parser.add_argument(
+            '--file',
+            type=str,
+            default=None)
+    run_parser.add_argument(
+            '--extra-vars',
+            nargs='*')
```

Comment:
What is prompting the extra-vars part of this anyway, is a desire to pass a version stamp into the ansible playbooks (for use when downloading docker images for example) from Jenkins, when the ansible-flow call is being made, rather than having to sed-modify the project.yml file to add the extra args.